### PR TITLE
Set up zoom later

### PR DESCRIPTION
**Pull request by ruiaraujo**
We are having a crash with the error

```
DOMException: Failed to execute 'inverse' on 'SVGMatrix': The matrix is not invertible.
    at SvgPanZoom.push../node_modules/svg-pan-zoom/src/svg-pan-zoom.js.SvgPanZoom.zoomAtPoint 
    at SvgPanZoom.push../node_modules/svg-pan-zoom/src/svg-pan-zoom.js.SvgPanZoom.zoom
    at SvgPanZoom.push../node_modules/svg-pan-zoom/src/svg-pan-zoom.js.SvgPanZoom.publicZoom
    at Object.zoom
    at Viewport.enableZoom
    at new Viewport
```

This "hack" prevents the issue.

### CODE REVIEW

File: src/graph/viewport.ts [hunk 1]
```diff
@@ -55,7 +57,8 @@ export class Viewport {
     this.zoomer.zoom(0.95);
   }
 
-  bindClick() {
+  
+  () {
```
Comment (IvanGoncharov) [inline src/graph/viewport.ts]: Is this change intentional?

Comment (ruiaraujo) [reply src/graph/viewport.ts]: nope, already removed.

**Comment by IvanGoncharov** [comment]
@ruiaraujo Thanks for PR 👍 
Merged and released as `1.0.0-rc.28` 🎉
